code/game/objects/items/hand_items.dm

```diff
@@ -127,7 +127,7 @@
 		return FALSE
 
 	var/obj/item/bodypart/head/the_head = target.get_bodypart(BODY_ZONE_HEAD)
-	if((target.get_biological_state() != BIO_FLESH_BONE && target.get_biological_state() != BIO_JUST_FLESH) || !IS_ORGANIC_LIMB(the_head))
+	if(!(the_head.biological_state & BIO_FLESH) || !IS_ORGANIC_LIMB(the_head))
```

Comment:
We check if the head isnt organic and if it isnt organic?
Why are we doing all these checks when IS_ORGANIC_LIMB exists?

Reply:
> I don't really understand why you make all these exceptions on piercing and slashwounds if the limb is from a NOBLOOD species, wouldn't it'd be easier to just not give noblood species the wound to begin with? The only difference seems to be some fluff message about holes in your body

This would break dismemberment for species without blood entirely (if you put a human limb on a skeleton for example, it would not then be able to be dismembered), which is a pretty serious issue that i want to avoid.

Reply:
Fuck i wanted to post this in general comments, my bad

Reply:
Ok so, about this
By default, limbs have biological state set to BIO_FLESH_BONE. Biological state is a different variable from the status variable, and does not imply that the limb is robotic or not, and i want to keep it like that. This check while seemingly absurd, is entirely correct, unless i merge the status variable and biological_state variable.

Reply:
Something tells me that doing that would be a pretty bad idea, so i'd rather not do that.